### ISSUE TITLE
java.lang.UnsatisfiedLinkError: [...] couldn't find "libwallet_core_all.so"

### ISSUE THREAD

**Issue by albert0m** [italia/pagopa-android-example#3]
After downloading the project (as is), I get the error:

`java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/it.pagopa.ppaexample-2/base.apk"],nativeLibraryDirectories=[/data/app/it.pagopa.ppaexample-2/lib/arm64, /data/app/it.pagopa.ppaexample-2/base.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64]]] couldn't find "libwallet_core_all.so"`

**Comment by gammam** [comment]
Did you downloaded master branch or Release 1.0.0 ?

**Comment by albert0m** [comment]
yep, that was it, I switched to the branch. Is there any way to mock a generic use case? I get a 404 when using the baseUrl https://acardste.vaservices.eu/

**Comment by gammam** [comment]
You need a valid idPayment in order to start a session.
In order to get a valid idPayment you can :
- be connected with a PA already connected to pagoPA test environment ( the idPayment is contained in _nodoInviaCarrelloRPT_ response)
- request a list of idPayment.

Unfortunally at this moment there isn't a defined process in order to address this request. Please use one of this 

f69806d1-e4b3-4505-87cc-f2adacdc2495
1c36187e-d81a-4804-b862-7db084d702a7
397c6f91-8dbe-4ac7-82ea-16d423e911bd
7d6df046-5d70-4286-b6f4-01e6ab4fdb5e
0c19b63f-d41a-4abc-915f-16859d2767c8
61748b7e-a632-47aa-bbf4-618c7cb89403
32b70153-a45f-4b53-ba1d-b481d80c2bd7
5177292d-0e64-407d-b25a-701cdecdb6c4
b80d2ab9-cb96-4d65-8322-e30808308671
77a2767c-373f-4732-a53a-5c0366a5f0b5
74f7eadd-d39c-4c00-8497-4571492453b8
8d0b7f0e-dfc9-4669-8968-3a28591db802
ce648dfe-b49a-427c-83f4-da30737ae9fc
8ee4e49c-9c92-465f-a5ba-71b1228aa024
b3a511b5-5bd3-4faa-8d6a-78bc55b9f058
2de870ce-5679-4a5b-8f58-8e7c499c5354
56d0b433-2909-4fcd-aea2-2fbf4611ee48
e8d31801-ffe3-4f15-bcb6-baec92483ac9
16fe0ec6-dc73-4acf-922d-e680d8689942
9f78ce8c-3e2f-4162-8241-1c32a23b5d73
27bd6b85-c52f-43ba-aa56-bc2495c27161
652b297f-6a5c-44e3-a38b-ad8caac83be0
